manifests/init.pp

```diff
@@ -15,6 +15,7 @@
   $minimum_version                                                    = $collectd::params::minimum_version,
   $package_ensure                                                     = $collectd::params::package_ensure,
   $package_install_options                                            = $collectd::params::package_install_options,
+  $package_keyserver                                                  = $collectd::params::package_keyserver,
```

Comment:
please add a datatype to the new param.

Reply:
I tipbased and added the `String` datatype to the package_keyserver parameter.

I was considering using `Stdlib::Fqdn` but is not supported by the minimum stdlib library supported by this module (`4.13.1`)